### ISSUE TITLE
[Youtube] The left side of the homepage menu buttons is cut off 

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#3102]
**[Affected Versions]:**
- Firefox Reality version 10-140931839 (a360c20).

**[Affected Platforms]:**
- HTC Vive Plus - Android 7.1.2
- Pico Neo 2 - Android - 8.1.0
- Oculus Quest - Android 7.1.1

**[Prerequisites]:**
- The headset is turned on and the app is opened.

**[Steps to reproduce]:**                                        
1. Navigate to Youtube homepage (https://www.youtube.com/).
2. Observe the website's menu buttons from the left side.

**[Expected results]:**
- The buttons are correctly displayed.

**[Actual results]:**
- The left side of the buttons is cut off.

**[Additional Notes]:**
- This issue is reproducible on all sizes where the menu is displayed (1x size and up).
- It could be a Youtube related change since this issue is now also reproducible in the production (v9) off the app.
- Attached is a screenshot of the issue:
![org mozilla vrbrowser-20200403-151729](https://user-images.githubusercontent.com/40061643/78360874-213c6c00-75c0-11ea-86ac-061c8834a8ce.jpg)

**Comment by keianhzo** [comment]
@Softvision-RemusDranca I can't reproduce this anymore, Can you confirm it still happens and that the STRs are still valid?

**Comment by Softvision-RemusDranca** [comment]
I've retested this issue using the same steps and it's no longer reproducible with the Firefox Reality version 10 RC5 (e84d3c9). The left side of the homepage menu buttons on Youtube website are now correctly displayed.
Considering this I will close this issue as verified fixed.

Tested on HTC Vive Focus (Android 7.1.2), Oculus Quest (Android 7.1.1), and Pico Neo 2 (Android - 8.1.0) devices.
